api/services/ConfigService.js

```diff
@@ -0,0 +1,176 @@
+ * If default value is an object, the environment variable is expected to be a
+ * serialized JSON object.
+ * If it's an array, it's expected to be a serialized JSON array.
+ * Otherwise, it returns the defaultValue.
+ *
+ * @method nanocloudConfigValue
+ * @private
+ * @param {String} name Name of the config variable
+ * @param {String} defaultValue The value to return if the environment variable
+ * "name" is not set.
+ * @return {Object} The environment variable if found (casted to the type of
+ * defaultValue). defaultValue otherwise.
+ */
+function nanocloudConfigValue(name, defaultValue) {
+  let value;
+
+  if (process.env.hasOwnProperty(name)) {
+    let type;
+
+    let value = process.env[name];
+
+    if (Array.isArray(defaultValue)) {
+      type = 'array';
+    } else {
+      type = (typeof defaultValue);
+    }
+
+    switch (type) {
+      case 'number':
+        value = parseInt(value, 10);
+        if (Number.isNaN(value)) {
+          throw new Error(`Config variable '${name}' must be an number.`);
+        }
+        break;
+
+      case 'array':
+        value = JSON.parse(value);
+        if (!Array.isArray(value)) {
+          throw new Error(`Config variable '${name}' must be an array.`);
+        }
+        break;
+
+      case 'object':
+        value = JSON.parse(value);
+        if (typeof value !== 'object') {
+          throw new Error(`Config variable '${name}' must be an object.`);
+        }
+        break;
+
+      case 'boolean':
+        if (value === 'true') {
+          value = true;
+        } else if (value === 'false') {
+          value = false;
+        }
+        throw new Error(`Config variable '${name}' must be an boolean.`);
+
+   // case 'string':
+   //   value = value;
+   //   break;
+
+    }
+  } else {
+    value = defaultValue;
+  }
+
+  return value;
+}
+
+/**
+ * get retreives the values of the specified config variables.
+ *
+ * @method get
+ * @public
+ * @param {[]String} keys The names of the variable to retreive
+ * @return {Promise[Object]} A promise that resolves a hash of the retreived
+ * variables
+ */
+function get(...keys) {
+  return new Promise((resolve, reject) => {
+    Config.find({
+      key: keys
+    }, (err, res) => {
+      if (err) {
+        reject(err);
+      } else {
+        let rt = {};
+        res.forEach((row) => {
+          rt[row.key] = row.value;
+        });
+        resolve(rt);
+      }
+    });
+  });
+}
+
+/**
+ * set saves the specified configuration variable in the database.
+ * If the variable exists already, the value is updated.
+ *
+ * @method set
+ * @public
+ * @param {String} key The name of the variable to create
+ * @param {String} value The value of the variable to create
+ * @return {Promise[null]}
+ */
+function set(key, value) {
+  return new Promise((resolve, reject) => {
+    Config.query({
+      text: `INSERT INTO
+      config ("key", "value", "created-at", "updated-at")
```

Comment:
`created-at` and `updated-at` must be formatted in camelCase to follow default sails behavior.
These fields should only be formatted in *kebab-case* in json-api payloads to follow this recommendation: http://jsonapi.org/recommendations/#naming